README.md

```diff
@@ -61,6 +61,10 @@ yarn build
 
 ### Testing
 
+To run the tests independently of the existing backend, run `yarn test-serve` / `npm run test-serve`
+
+DISCLAIMER: In Footer test file, "valid link to github" test may fail due to transient failures.
```

Comment:
Please make this more user-centered by listing exactly the report from `yarn test` that appears when this particular test  case fails.

Reply:
You might need to temporarily reduce the timeout to induce the failure and obtain the exact error. Does that make sense?

Reply:
We actually couldn't replicate the error a much as we tried so turns out we probably don't need the disclaimer!